libraries/AP_Common/AP_Time.h

```diff
@@ -0,0 +1,16 @@
+/*
+ * AP_Time.h
+ */
+
+#pragma once
+
+#include <AP_Common/AP_Common.h>
+#include <time.h>
```

Comment:
I see that @khancyr suggested to put this here, but I don't see why. We should put includes in cpp files as much as we can.